docs/en/about-appium/platform-support.md

```diff
@@ -13,82 +13,86 @@ This section lists drivers that are supported by Appium team.
   is effected using a bundled [Chromedriver](http://chromedriver.chromium.org)
   server as a proxy. With 4.3, automation works on official Chrome
   browser or Chromium only. With 4.4+, automation also works on the built-in
   "Browser" app. Chrome/Chromium/Browser must already be installed on the
   device under test. See the [mobile web doc](/docs/en/writing-running-appium/web/mobile-web.md) for instructions.
-* Hybrid support: Yes. See the [hybrid doc](/docs/en/writing-running-appium/web/hybrid.md) for instructions.
-  * With default Appium automation backend: versions 4.4 and up
-* Support for automating multiple apps in one session: Yes
-* Support for automating multiple devices simultaneously: Yes,
+- Hybrid support: Yes. See the [hybrid doc](/docs/en/writing-running-appium/web/hybrid.md) for instructions.
+  - With default Appium automation backend: versions 4.4 and up
+- Support for automating multiple apps in one session: Yes
+- Support for automating multiple devices simultaneously: Yes,
   though Appium must be started using different ports for the server
-   parameters `--port`, `--bootstrap-port` and/or
+  parameters `--port`, `--bootstrap-port` and/or
   `--chromedriver-port`. See the [server args doc](/docs/en/writing-running-appium/server-args.md) for more
   information on these parameters.
-* Support for automating vendor-provided or third-party apps: Yes
-* Support for automating custom, non-standard UI controls: No
+- Support for automating vendor-provided or third-party apps: Yes
+- Support for automating custom, non-standard UI controls: No
 
 ### macOS Support
 
 macOS automation is supported with below drivers.
 
-* The [Mac2Driver](/docs/en/drivers/mac2.md) is for macOS 10.15 or later
-* The (deprecated) [MacDriver](/docs/en/drivers/mac.md) is for lower macOS versions
+- The [Mac2Driver](/docs/en/drivers/mac2.md) is for macOS 10.15 or later
+- The (deprecated) [MacDriver](/docs/en/drivers/mac.md) is for lower macOS versions
 
 ### Windows Desktop Support
 
 Windows automation is supported with below drivers.
 
-* The [WinAppDriver](/docs/en/drivers/windows.md)
-* The [geckodriver](/docs/en/drivers/gecko.md) for Firefox and [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView)
+- The [WinAppDriver](/docs/en/drivers/windows.md)
+- The [geckodriver](/docs/en/drivers/gecko.md) for Firefox and [GeckoView](https://wiki.mozilla.org/Mobile/GeckoView)
 
 ## Vendors/Community based drivers
 
 This section lists drivers that are supported by vendors and community.
 
 ### You.i Engine Support
 
-* The [You.i Engine](https://github.com/YOU-i-Labs/appium-youiengine-driver)
+- The [You.i Engine](https://github.com/YOU-i-Labs/appium-youiengine-driver)
 
 ### Flutter Support
 
-* The [Flutter Driver](https://github.com/truongsinh/appium-flutter-driver)
+- The [Flutter Driver](https://github.com/truongsinh/appium-flutter-driver)
+
+### Roku Support
+
+- The [Roku Driver](https://github.com/cbuelvasc/appium-roku-driver)
```

Comment:
It makes sense to mention in the driver readme is not included by default and should be installed separately from NPM in order to make it working

Reply:
It's update:  https://github.com/cbuelvasc/appium-roku-driver

# 🚀 Appium Roku Driver

[![NPM version](http://img.shields.io/npm/v/appium-roku-driver.svg)](https://npmjs.org/package/appium-roku-driver)
[![Downloads](http://img.shields.io/npm/dm/appium-roku-driver.svg)](https://npmjs.org/package/appium-roku-driver)
[![Build Status](https://api.travis-ci.com/cbuelvasc/appium-roku-driver.svg?token=6pPhwGxQvWEfxb4ynd76&branch=master)](https://api.travis-ci.com/cbuelvasc/appium-roku-driver)

Appium Roku Driver is a test automation tool for Roku devices. Appium Roku Driver automates Roku applications, tested on real devices. Appium Roku Driver is part of the [Appium](https://github.com/appium/appium) test automation tool.

## Installation

In order to use `appium-roku-driver`, we need to use `appium` version `1.16.0` or higher

```
npm i -g appium-roku-driver
```

Reply:
Some APIs listed in the readme seems to have its own commands (not available via Appium [route](https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js)), so it is probably helpful to address they are available only via JS (when a user imports RokuDriver)